go/updater/updater.go

```diff
@@ -101,6 +101,18 @@ func (u *Updater) checkForUpdate(skipAssetDownload bool, force bool, requested b
 		return
 	}
 
+	// do only if instructions are empty/nil
```

Comment:
This comment is kind of obvious from the code? Maybe a better comment is:

// Update instruction might be empty, if so we'll fill in with the default instructions for the platform